### ISSUE TITLE
Multiple declarations of dictionaries for Phase2Tracker* classes

### ISSUE THREAD

**Issue by Dr15Jones** [cms-sw/cmssw#14644]
The IBs are showing a warning that we have mulitple shared libraries containing the same dictionaries:
https://cmssdt.cern.ch/SDT/cgi-bin/buildlogs/slc6_amd64_gcc493/CMSSW_8_1_X_2016-05-25-2300/pyRelValMatrixLogs/run/140.4_ZEEMM_13_HI+ZEEMM_13_HIINPUT+DIGIHI+RECOHI+HARVESTHI/step3_ZEEMM_13_HI+ZEEMM_13_HIINPUT+DIGIHI+RECOHI+HARVESTHI.log

```
Warning in <TInterpreter::ReadRootmapFile>: class  edm::Ref<edmNew::DetSetVector<Phase2TrackerCluster1D>,Phase2TrackerCluster1D,edmNew::DetSetVector<Phase2TrackerCluster1D>::FindForDetSetVector> found in libDataFormatsPhase2TrackerCluster.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edm::Ref<edmNew::DetSetVector<Phase2TrackerCluster1D>,Phase2TrackerCluster1D> found in libDataFormatsPhase2TrackerCluster.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edm::Wrapper<edm::Ref<edmNew::DetSetVector<Phase2TrackerCluster1D>,Phase2TrackerCluster1D,edmNew::DetSetVector<Phase2TrackerCluster1D>::FindForDetSetVector> > found in libDataFormatsPhase2TrackerCluster.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edm::Wrapper<edm::Ref<edmNew::DetSetVector<Phase2TrackerCluster1D>,Phase2TrackerCluster1D> > found in libDataFormatsPhase2TrackerCluster.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  Phase2TrackerRecHit1D found in libDataFormatsPhase2TrackerRecHit.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edm::Wrapper<edmNew::DetSet<Phase2TrackerRecHit1D> > found in libDataFormatsPhase2TrackerRecHit.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edm::Wrapper<edmNew::DetSetVector<Phase2TrackerRecHit1D> > found in libDataFormatsPhase2TrackerRecHit.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edm::Wrapper<std::vector<edmNew::DetSet<Phase2TrackerRecHit1D> > > found in libDataFormatsPhase2TrackerRecHit.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edm::Wrapper<vector<edmNew::DetSet<Phase2TrackerRecHit1D> > > found in libDataFormatsPhase2TrackerRecHit.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edmNew::DetSet<Phase2TrackerRecHit1D> found in libDataFormatsPhase2TrackerRecHit.so  is already in libDataFormatsTrackerRecHit2D.so 
Warning in <TInterpreter::ReadRootmapFile>: class  edmNew::DetSetVector<Phase2TrackerRecHit1D> found in libDataFormatsPhase2TrackerRecHit.so  is already in libDataFormatsTrackerRecHit2D.so 
```

**Comment by Dr15Jones** [comment]
assign reconstruction

**Comment by slava77** [comment]
It's IB issue
we need a full build

**Comment by slava77** [comment]
@smuzaffar @degano please trigger a full build in 81X
we should check after that if the duplicates are gone (they should, the reason is a move of files/classes)

**Comment by smuzaffar** [comment]
Next 81X IBs (23h00) should be full IBs. I am anyway triggering a 17h00 IB for gcc530 arch now

**Comment by slava77** [comment]
+1

fixed in  #15183
